### PR TITLE
Tessa/remove assets

### PR DESCRIPTION
**Pull request by tesk9**
While working on something else, I noticed that we were still passing an assets record to a few styleguide examples. This PR aims to fix that, so that we're only passing assets where it's unavoidable not to.

This required a new version of segmented control. As far as I can tell, SegmentedControl only uses Icons on the rate results page, so I focused on getting the segmented control example to look nice with Svg icons that correspond to the rate results view. 

Screen Shot 2019-10-24 at 12 29 25 PM

<details>
<summary>Rate results view, for reference
</summary>

Screen Shot 2019-10-24 at 11 03 11 AM

</details>

<details>
<summary>Styleguide showing icons prior to this change
</summary>

Screen Shot 2019-10-24 at 11 03 06 AM

</details>

I also had to convert some old icons to elm-svg to get this to work. However, it kinda seems like the svg code I exported from zeplin wasn't quite right. I had to futz with the feColorMatrix for the "star" icon to get it to look darker on the edges, and then the empty star seemed at first like the viewbox was wrong, but I think actually the path is getting cut off. @NoRedInk/design are there other icons I should use? 

Screen Shot 2019-10-24 at 12 42 20 PM

### CODE REVIEW

**Comment by bendansby** [comment]
Yay for removing PNGs! I'll defer to @staceyadams on the details

File: src/Nri/Ui/Svg/V1.elm [hunk 1]
```diff
@@ -1,24 +1,33 @@
 module Nri.Ui.Svg.V1 exposing
     ( Svg
+    , withColor
```
Comment (michaelglass) [inline src/Nri/Ui/Svg/V1.elm]: 😍

File: styleguide-app/Examples/Button.elm [hunk 2]
```diff
@@ -108,26 +107,13 @@ init assets =
...
-        , ( "Just Lock"
-          , Icon.lock assets
-                |> Icon.decorativeIcon
-                |> NriSvg.fromHtml
-                |> Just
-                |> Control.value
-          )
+        [ ( "Nothing", Control.value Nothing )
+        , ( "Just Performance", Control.value (Just UiIcon.performance) )
+        , ( "Just Share", Control.value (Just UiIcon.share) )
```
Comment (michaelglass) [inline styleguide-app/Examples/Button.elm]: 🤘

File: styleguide-app/Examples/UiIcon.elm [hunk 3]
```diff
@@ -58,5 +58,10 @@ example =
             , ( "attention", UiIcon.attention )
             , ( "exclamation", UiIcon.exclamation )
             ]
+        , (IconExamples.view "Notifs" << List.map (Tuple.mapSecond Svg.toHtml))
```
Comment (michaelglass) [inline styleguide-app/Examples/UiIcon.elm]: 👍

**Pull request merged**